LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Regions/RegionCN470.cs

```diff
@@ -69,6 +69,14 @@ public RegionCN470()
             MaxADRDataRate = 7;
             RegionLimits = new RegionLimits((min: 470.3, max: 509.7), validDatarates, validDatarates, 0, 0);
 
+            UpstreamAndDownstreamJoinFrequencies = new List<(double, double)>
```

Comment:
Is there a reason we using a list here, Wouldn't a dictionary perform better?

Reply:
No because the frequencies need to be in the given order as they map to the channel number (which is the index in the list).